resources/js/Components/Form/EntryCreateForm.tsx

```diff
@@ -304,7 +311,9 @@ const EntryCreateForm = ({
                 <Button
                     fullWidth
                     onClick={() =>
-                        type === 'Create' ? setActiveStep(1) : setActiveStep(0)
+                        type === 'Edit' || data.dungeon_master === ''
```

Comment:
you can just use `!data.dungeon_master` if you are looking for empty string, undefined or null

Reply:
Done!